src/app/components/Document/__snapshots__/index.test.jsx.snap

```diff
@@ -41,9 +41,9 @@ exports[`Document should render correctly 1`] = `
       href="https://www.bbc.com/news/articles/c9rpqy7pmypo"
       rel="canonical"
     />
-    <style>
-      html { color: red; }
-    </style>
+    <style
+      data-styled-components=""
```

Comment:
Phil pointed out the `<style>` tag probably shouldn't be added if we have no styles (agreed 👍 )
And that we need a snapshot with styles added (agreed 👍 )

Reply:
1. feels like a premature optimisation (how often will we have _no_ styles?) - so ignored.
2. I've re-added the inline styles test in https://github.com/BBC-News/simorgh/pull/921/commits/7c87b10045500c1ed4d8ec94d69bcd2e2aa0c2d8

But now it looks like we need a snapshot test for AMP, where instead of `data-styled-components` we have `amp-custom` as an attribute.

Reply:
Blocked on #958. (Want to test AMP snapshot).

Reply:
#958 is merged now. Ready to code review / merge.